hermes/usecases/fortune.py

```diff
@@ -0,0 +1,25 @@
+from hermes.config import settings
+from hermes.schemas.fortune import FortuneSeafarerSchema
+from hermes.services.fortune import FortuneClient
+from hermes.usecases.common import parse_odata_response
+
+
+def get_ws_seaman_list() -> list[FortuneSeafarerSchema]:
+    if settings.environment_type in ["LOCAL", "DEVELOPMENT", "STAGING"]:
```

Comment:
```suggestion
    if settings.environment_type in [ "DEVELOPMENT", "STAGING"]:
```
we dont need local right?